free-programming-books.md

```diff
@@ -1387,6 +1394,7 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Core Servlets and JavaServer Pages, 2nd Ed. (2003)](http://pdf.coreservlets.com) - Marty Hall and Larry Brown
 * [Data Structures in Java for the Principled Programmer (2007)](http://dept.cs.williams.edu/~bailey/JavaStructures/Book_files/JavaStructures.pdf) - Duane A. Bailey (PDF)
 * [Google's Java Style Guide](https://google.github.io/styleguide/javaguide.html)
+* [Introduction to Computer Science](https://introcs.cs.princeton.edu/java/cs/) - by Robert Sedgewick and Kevin Wayne (HTML)
```

Comment:
This "booksite" is not the book. We can include it if you think it's still useful, but should add a notation.

Reply:
Nowadays Books exists in some formats. This web page IS the book (in HTML form hence the (HTML) instead of (PDF) or (OTHER FORMAT))

Reply:
By the way i have a question
Suppose there is an author's  page in which  other than author's textbook refer other useful informations like exercises other subject matter related links bibliography etc 
In that case IMHO is more useful/valuable refer in the books list the author's page(that contains the author's PDF) than the author's textbook link
Such typical use cases are Uni's profs pages

Reply:
We have lots of HTML websites-as-books. This isn't one of those. Its a free-website-to-go-with-book-that-you've-bought. Nothing wrong with that, but it's not the not-free-book-that-you-buy.

Reply:
We often link to a book webpage where links to the book file are presented along with related material.

Reply:
> We have lots of HTML websites-as-books. This isn't one of those. Its a free-website-to-go-with-book-that-you've-bought. Nothing wrong with that, but it's not the not-free-book-that-you-buy.

Right now i am in house and have the book at hand. I was pretty sure about that. The prof page is exactly the book more in their page is much more better organized than in printed form

Reply:
> We often link to a book webpage where links to the book file are presented along with related material.

That's the right thing i guess :+1: